cmd/job/run_input.go

```diff
@@ -0,0 +1,205 @@
+package job
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/odpf/salt/log"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	pb "github.com/odpf/optimus/api/proto/odpf/optimus/core/v1beta1"
+	"github.com/odpf/optimus/cmd/connectivity"
+	"github.com/odpf/optimus/cmd/logger"
+	"github.com/odpf/optimus/config"
+	"github.com/odpf/optimus/models"
+	"github.com/odpf/optimus/utils"
+)
+
+const (
+	jobRunInputCompileAssetsTimeout = time.Minute * 1
+
+	taskInputDirectory = "in"
+	unsubstitutedValue = "<no value>"
+)
+
+type jobRunInputCommand struct {
+	logger       log.Logger
+	clientConfig *config.ClientConfig
+
+	assetOutputDir string
+	runType        string
+	runName        string
+	scheduledAt    string
+
+	keysWithUnsubstitutedValue []string
+}
+
+// NewJobRunInputCommand gets compiled assets required for a job run
+func NewJobRunInputCommand(clientConfig *config.ClientConfig) *cobra.Command {
```

Comment:
what is the response when job run doesn't exist

Reply:
`Error: request failed for job <job_name>: rpc error: code = Internal desc = resource not found: failed to get JobRun by ScheduledAt for job <job_name>`